tests/core/contracts/test_contract_constructor_encoding.py

```diff
@@ -20,6 +20,19 @@ def test_contract_constructor_abi_encoding_with_constructor_with_no_args(SimpleC
     assert deploy_data == SIMPLE_CONSTRUCTOR_CODE
 
 
+@pytest.mark.parametrize(
+    'args,kwargs',
+    (
+        (None, 'kwarg-is-ignored'),
+        ('arg-is-ignored', None),
+    ),
+)
+def test_contract_error_if_additional_args_are_supplied_with_no_constructor_fn(MathContract,
+                                                                               args, kwargs):
+    with pytest.raises(TypeError):
```

Comment:
Can you tighten this test a little by adding `match="Constructor args"` to the `pytest.raises` call so that we're sure it's matching the correct error.

Reply:
good call, fixed & repushed